site/content/docs/01-component-format.md

```diff
@@ -24,7 +24,7 @@ All three sections — script, styles and markup — are optional.
 
 A `<script>` block contains JavaScript that runs when a component instance is created. Variables declared (or imported) at the top level are 'visible' from the component's markup. There are four additional rules:
 
-##### 1. `export` creates a component prop
+#### props
```

Comment:
I'm not sure the new headings follow from the previous sentence: "There are four additional rules"

Reply:
ok. I've removed that sentence since it isn't really necessary